packages/e2e-tests/specs/site-editor/style-variations.test.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * WordPress dependencies
+ */
+import { trashAllPosts, activateTheme } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { siteEditor } from './utils';
```

Comment:
In a recent PR, these utils has been moved to e2e-test-utils, so a rebase might break things here.